src/widget.tsx

```diff
@@ -61,6 +65,7 @@ export class LuxWidgetView extends DOMWidgetView {
       currentVis:object,
       recommendations:any[],
       intent:string,
+      intentIndex: object,
```

Comment:
This is the index for the selected vis? Can we give this a more interpretable name?

Reply:
changed to selectedIntentIndex